src/edge.hal-interface-adapter/src/halInterfaceAdapter.js

```diff
@@ -148,10 +158,7 @@ module.exports = class HalInterfaceAdapter {
     async __onHalMessageReceive(msg, topic) {
         this.logger.debug(`Received HAL message ${JSON.stringify(msg)} for topic ${topic}`);
 
-        // Receives hal/10.CruiseStatus2
-        // Strip namespace
-        topic = topic.replace(this.halBroker.topicNs, '');
-
+        // Receives 10.CruiseStatus2
```

Comment:
Should this comment still be here? Looks like it was part of a debugging effort.

Reply:
I think it is rather a descriptive comment what input is handled by this method. In previous version of the halInterfaceAdapter.js it is ''Receives hal/10.CruiseStatus2'.

I'll improve it a bit.